src/components/ui/navigation-menu.tsx

```diff
@@ -35,7 +35,7 @@ NavigationMenuList.displayName = NavigationMenuPrimitive.List.displayName
 const NavigationMenuItem = NavigationMenuPrimitive.Item
 
 const navigationMenuTriggerStyle = cva(
-  "group inline-flex h-10 w-max items-center justify-center rounded-md bg-background px-4 py-2 text-sm font-medium transition-colors hover:bg-accent hover:text-accent-foreground focus:bg-accent focus:text-accent-foreground focus:outline-none disabled:pointer-events-none disabled:opacity-50 data-[active]:bg-accent/50 data-[state=open]:bg-accent/50",
```

Comment:
This is the class that would add a white background to the header buttons which doesn't make a lot of sense stylistically

Reply:
Background is an intended behaviour, see https://github.com/Schroedinger-Hat/schroedinger-hat-website/pull/145#issuecomment-2523443281

Reply:
If it's up for discussion, I still find this behaviour not on par with the rest of the website, so would push for removal of bg @LobeTia @TheJoin95

Reply:
I don't like the background that much but if it's for a fallback on smaller screen maybe we can find a hybrid solution with media queries

Reply:
Smaller screen in this case is 1920x1080, so not small. I have a strong opinion on this one. Let's remove the bg, please

Reply:
I feel this is an example of the law of triviality.

The white background on menu items ensures readability, especially where the BlurredBackground's random gradients size/positions may interfere. Removing it doesn't improve functionality or accessibility and risks reducing usability for color-blind users.

Design decisions must be based on usability, not subjective taste. As I curate the site's art direction, this change will not proceed without a valid argument beyond the important, but relative, personal preferences.

Further discussion on the background can happen in a dedicated PR. This PR will only be evaluated for its ability to fix tablet layouts for homepage.